docs/intro-react.md

```diff
@@ -355,8 +469,8 @@ import React, { useState } from 'react';
 
 Then you declare the component’s state by calling `useState` inside its function. In this example, `useState` creates an `isHungry` state variable:
 
-```jsx
-const Cat = (props) => {
+```tsx
```

Comment:
Going to take a separate pass through inline jsx later to change to tsx. Though they cannot be linted/typechecked since these are not full modules.